Modules/Module_Import/Import/xASL_imp_DCM2NII_Subject_SortASLVolumes.m

```diff
@@ -146,6 +146,21 @@
         elseif  ~isempty(strfind(scan_name,'M0'))
             [nii_files,ASLContext] = xASL_bids_MergeNifti(nii_files, 'M0', niiTable);
         end
+	else
+		% In case of a single file, we need to check issues with orientation
+		% Check for a special case of Siemens MOSAIC files without Phoenix field
+		if isfield(tmpJSON, 'Manufacturer') && strcmpi(tmpJSON.Manufacturer, 'Siemens') && isfield(tmpJSON, 'ImageType') && ~isempty(regexpi(tmpJSON.ImageType, 'MOSAIC')) && (~isfield(tmpJSON, 'PhoenixProtocol') || isempty(tmpJSON.PhoenixProtocol))
```

Comment:
For readability, I would put the if cases on separate rows:

if isfield(tmpJSON, 'Manufacturer') && strcmpi(tmpJSON.Manufacturer, 'Siemens')
% this happens only for Siemens scans

if isfield(tmpJSON, 'ImageType') && ~isempty(regexpi(tmpJSON.ImageType, 'MOSAIC'))
% Siemens MOSAIC scans specifically

if ~isfield(tmpJSON, 'PhoenixProtocol') || isempty(tmpJSON.PhoenixProtocol)
% Siemens MOSAIC scans that have no Phoenix protocol stored in the DICOM header

Reply:
I've separated it to four lines. But I avoided doing several IFs cause this creates too much ifs, ends, and too many line indents...